config.py

```diff
@@ -164,47 +166,6 @@ class Config:
             "max_builds": 5,
             "build_timeout": 150,  # TODO: revisit
         },
-        "ec2-t3-xlarge-us-east-2": {
-            "info": "Supported benchmark langs: Python, R, JavaScript",
-            "default_filters": {
-                "arrow-commit": {
-                    "langs": {
-                        "Python": {
-                            "names": [
-                                "dataset-filter",
-                                "dataset-read",
-                                "dataset-select",
-                                "file-read",
-                                "recursive-get-file-info",
-                                "wide-dataframe",
-                            ]
-                        },
-                        "R": {
-                            "names": [
-                                "dataframe-to-table",
-                                "file-read",
-                                "partitioned-dataset-filter",
-                            ]
-                        },
-                        "JavaScript": {"names": ["js-micro"]},
-                        "C++": {"names": ["cpp-micro"]},
-                    }
-                },
-                "benchmarkable-repo-commit": {
```

Comment:
Need to keep this somewhere?

Reply:
I don't think so; this was leftover from testing